carbonmark/components/Logos/CarbonmarkLogoBlue.tsx

```diff
@@ -0,0 +1,86 @@
+export const CarbonmarkLogoBlue = () => (
```

Comment:
This is unfortunate :( is it so that you have control over the color/css properties?

Reply:
So this will be removed with v2. This is only added to keep the homepage icon as it was before the changes to the navigation header. In v2, we will be reusing the component used on the marketplace navigation.